core/serial_command.py

```diff
@@ -810,33 +763,33 @@ def select_channel(self, channel):
 
 def command_select_channel(par):
     """Get the command to select a single channel"""
-    command = _cmd_channel_select_single_start+chr(par)+_cmd_channel_select_single_end
-    buffer_check = "B"+str((int(par)-1)/8+1)+_cmd_channel_select_single_end
+    command = p._cmd_channel_select_single_start+chr(par)+_cmd_channel_select_single_end
+    buffer_check = p._cmd_disable_ext_trig+str((int(par)-1)/8+1)+_cmd_channel_select_single_end
```

Comment:
again, this one (and the line above), I think you're missing a couple of p's. _cmd_channel_select... isn't currently being imported from the parameters file